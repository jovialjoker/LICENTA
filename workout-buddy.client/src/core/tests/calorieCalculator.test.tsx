import { render, fireEvent } from "@testing-library/react";
import { screen } from "@testing-library/dom";
import CaloriesCalculator from "../pages/calorieCalculator/CalorieCalculator";

const { ResizeObserver } = window;

beforeEach(() => {
  window.ResizeObserver = jest.fn().mockImplementation(() => ({
    observe: jest.fn(),
    unobserve: jest.fn(),
    disconnect: jest.fn(),
  }));
});

afterEach(() => {
  window.ResizeObserver = ResizeObserver;
  jest.restoreAllMocks();
});

test("renders calculator with no calories(empty) for first render", () => {
  render(<CaloriesCalculator />);
  const calorieText = screen.getByTestId("calorie-text");

  expect(calorieText.textContent).toBe("Recommendation calories ");
});

test("renders calculator with 0 value for no input", () => {
  render(<CaloriesCalculator />);
  const calorieText = screen.getByTestId("calorie-text");
  const calorieButton = screen.getByTestId("calorie-button");

  fireEvent.click(calorieButton);

  expect(calorieText.textContent).toBe("Recommendation calories 0");
});

test("check calorie calculator snapshot", () => {
  const component = render(<CaloriesCalculator />);

  expect(component).toMatchSnapshot();
});
