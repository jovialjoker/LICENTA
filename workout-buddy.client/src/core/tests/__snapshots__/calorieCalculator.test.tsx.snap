// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check calorie calculator snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="chakra-container css-fn8iib"
      >
        <div
          class="chakra-stack css-tl3ftk"
        >
          <h2
            class="chakra-heading css-0"
          >
            Calorie calculator
          </h2>
          <h2
            class="chakra-heading css-i4e4a6"
          >
            Calculate your calories
          </h2>
          <p
            class="chakra-text css-i4e4a6"
            data-testid="calorie-text"
          >
            Recommendation calories 
          </p>
        </div>
        <div
          class="chakra-stack css-pua1y5"
        >
          <div
            class="chakra-form-control css-d85vxf"
            role="group"
          >
            <label
              class="chakra-form__label css-u9c2cp"
              for="field-:rg:"
              id="field-:rg:-label"
            >
              Gender
            </label>
            <div
              class=" css-b62m3t-container"
            >
              <span
                class="css-1f43avz-a11yText-A11yText"
                id="react-select-8-live-region"
              />
              <span
                aria-atomic="false"
                aria-live="polite"
                aria-relevant="additions text"
                class="css-1f43avz-a11yText-A11yText"
              />
              <div
                class=" css-13cymwt-control"
              >
                <div
                  class=" css-1fdsijx-ValueContainer"
                >
                  <div
                    class=" css-1jqq78o-placeholder"
                    id="react-select-8-placeholder"
                  >
                    Select...
                  </div>
                  <div
                    class=" css-qbdosj-Input"
                    data-value=""
                  >
                    <input
                      aria-autocomplete="list"
                      aria-describedby="react-select-8-placeholder"
                      aria-expanded="false"
                      aria-haspopup="true"
                      autocapitalize="none"
                      autocomplete="off"
                      autocorrect="off"
                      class=""
                      id="react-select-8-input"
                      role="combobox"
                      spellcheck="false"
                      style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                      tabindex="0"
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class=" css-1hb7zxy-IndicatorsContainer"
                >
                  <span
                    class=" css-1u9des2-indicatorSeparator"
                  />
                  <div
                    aria-hidden="true"
                    class=" css-1xc3v61-indicatorContainer"
                  >
                    <svg
                      aria-hidden="true"
                      class="css-tj5bde-Svg"
                      focusable="false"
                      height="20"
                      viewBox="0 0 20 20"
                      width="20"
                    >
                      <path
                        d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="chakra-form-control css-1dx7k9g"
            role="group"
          >
            <label
              class="chakra-form__label css-uvho5g"
              for="field-:rh:"
              id="field-:rh:-label"
            >
              Age
            </label>
            <input
              class="chakra-input css-124be3e"
              id="field-:rh:"
              type="number"
              value=""
            />
          </div>
          <div
            class="chakra-form-control css-1p8wjmg"
            role="group"
          >
            <label
              class="chakra-form__label css-u9c2cp"
              for="field-:ri:"
              id="field-:ri:-label"
            >
              Somatic type
            </label>
            <div
              class=" css-b62m3t-container"
            >
              <span
                class="css-1f43avz-a11yText-A11yText"
                id="react-select-9-live-region"
              />
              <span
                aria-atomic="false"
                aria-live="polite"
                aria-relevant="additions text"
                class="css-1f43avz-a11yText-A11yText"
              />
              <div
                class=" css-13cymwt-control"
              >
                <div
                  class=" css-1fdsijx-ValueContainer"
                >
                  <div
                    class=" css-1jqq78o-placeholder"
                    id="react-select-9-placeholder"
                  >
                    Select...
                  </div>
                  <div
                    class=" css-qbdosj-Input"
                    data-value=""
                  >
                    <input
                      aria-autocomplete="list"
                      aria-describedby="react-select-9-placeholder"
                      aria-expanded="false"
                      aria-haspopup="true"
                      autocapitalize="none"
                      autocomplete="off"
                      autocorrect="off"
                      class=""
                      id="react-select-9-input"
                      role="combobox"
                      spellcheck="false"
                      style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                      tabindex="0"
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class=" css-1hb7zxy-IndicatorsContainer"
                >
                  <span
                    class=" css-1u9des2-indicatorSeparator"
                  />
                  <div
                    aria-hidden="true"
                    class=" css-1xc3v61-indicatorContainer"
                  >
                    <svg
                      aria-hidden="true"
                      class="css-tj5bde-Svg"
                      focusable="false"
                      height="20"
                      viewBox="0 0 20 20"
                      width="20"
                    >
                      <path
                        d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="chakra-form-control css-1elsowl"
            role="group"
          >
            <label
              class="chakra-form__label css-u9c2cp"
              for="field-:rj:"
              id="field-:rj:-label"
            >
              Objective
            </label>
            <div
              class=" css-b62m3t-container"
            >
              <span
                class="css-1f43avz-a11yText-A11yText"
                id="react-select-10-live-region"
              />
              <span
                aria-atomic="false"
                aria-live="polite"
                aria-relevant="additions text"
                class="css-1f43avz-a11yText-A11yText"
              />
              <div
                class=" css-13cymwt-control"
              >
                <div
                  class=" css-1fdsijx-ValueContainer"
                >
                  <div
                    class=" css-1jqq78o-placeholder"
                    id="react-select-10-placeholder"
                  >
                    Select...
                  </div>
                  <div
                    class=" css-qbdosj-Input"
                    data-value=""
                  >
                    <input
                      aria-autocomplete="list"
                      aria-describedby="react-select-10-placeholder"
                      aria-expanded="false"
                      aria-haspopup="true"
                      autocapitalize="none"
                      autocomplete="off"
                      autocorrect="off"
                      class=""
                      id="react-select-10-input"
                      role="combobox"
                      spellcheck="false"
                      style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                      tabindex="0"
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class=" css-1hb7zxy-IndicatorsContainer"
                >
                  <span
                    class=" css-1u9des2-indicatorSeparator"
                  />
                  <div
                    aria-hidden="true"
                    class=" css-1xc3v61-indicatorContainer"
                  >
                    <svg
                      aria-hidden="true"
                      class="css-tj5bde-Svg"
                      focusable="false"
                      height="20"
                      viewBox="0 0 20 20"
                      width="20"
                    >
                      <path
                        d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="chakra-form-control css-0"
            role="group"
          >
            <label
              class="chakra-form__label css-u9c2cp"
              for="field-:rk:"
              id="field-:rk:-label"
            >
              Weight
            </label>
            <input
              class="chakra-input css-0"
              id="field-:rk:"
              type="number"
              value="0"
            />
          </div>
          <div
            class="chakra-form-control css-0"
            role="group"
          >
            <h2
              class="chakra-heading css-0"
            >
              Question 2
            </h2>
            <p
              class="chakra-text css-0"
            >
              On a scale of 1 to 10, how active are you on a day to day basis?
            </p>
            <div
              class="chakra-slider css-lnyvlk"
              style="position: relative; user-select: none; outline: 0; padding-top: 0px; padding-bottom: 0px;"
              tabindex="-1"
            >
              <div
                class="chakra-slider__track css-0"
                id="slider-track-:rm:"
                style="position: absolute; top: 50%; transform: translateY(-50%); width: 100%;"
              >
                <div
                  class="chakra-slider__filled-track css-mzpod1"
                  style="position: absolute; top: 50%; transform: translateY(-50%); width: 0%; left: 0%;"
                />
              </div>
              <div
                aria-orientation="horizontal"
                aria-valuemax="10"
                aria-valuemin="0"
                aria-valuenow="0"
                class="chakra-slider__thumb css-1hjihvm"
                id="slider-thumb-:rm:"
                role="slider"
                style="position: absolute; user-select: none; left: calc(0% - 0px);"
                tabindex="0"
              />
              <input
                type="hidden"
                value="0"
              />
            </div>
          </div>
          <div
            class="chakra-form-control css-0"
            role="group"
          >
            <h2
              class="chakra-heading css-0"
            >
              Question 3
            </h2>
            <p
              class="chakra-text css-0"
            >
              Are you doing any type of exercises?
            </p>
            <label
              class="chakra-checkbox css-1elxtiq"
            >
              <input
                aria-disabled="false"
                aria-invalid="false"
                class="chakra-checkbox__input"
                id="field-:rn:"
                style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
                type="checkbox"
                value="0"
              />
              <span
                aria-hidden="true"
                class="chakra-checkbox__control css-dnty2r"
              />
            </label>
          </div>
        </div>
        <button
          class="chakra-button css-4xx2wk"
          data-testid="calorie-button"
          type="button"
        >
          Calculate
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="chakra-container css-fn8iib"
    >
      <div
        class="chakra-stack css-tl3ftk"
      >
        <h2
          class="chakra-heading css-0"
        >
          Calorie calculator
        </h2>
        <h2
          class="chakra-heading css-i4e4a6"
        >
          Calculate your calories
        </h2>
        <p
          class="chakra-text css-i4e4a6"
          data-testid="calorie-text"
        >
          Recommendation calories 
        </p>
      </div>
      <div
        class="chakra-stack css-pua1y5"
      >
        <div
          class="chakra-form-control css-d85vxf"
          role="group"
        >
          <label
            class="chakra-form__label css-u9c2cp"
            for="field-:rg:"
            id="field-:rg:-label"
          >
            Gender
          </label>
          <div
            class=" css-b62m3t-container"
          >
            <span
              class="css-1f43avz-a11yText-A11yText"
              id="react-select-8-live-region"
            />
            <span
              aria-atomic="false"
              aria-live="polite"
              aria-relevant="additions text"
              class="css-1f43avz-a11yText-A11yText"
            />
            <div
              class=" css-13cymwt-control"
            >
              <div
                class=" css-1fdsijx-ValueContainer"
              >
                <div
                  class=" css-1jqq78o-placeholder"
                  id="react-select-8-placeholder"
                >
                  Select...
                </div>
                <div
                  class=" css-qbdosj-Input"
                  data-value=""
                >
                  <input
                    aria-autocomplete="list"
                    aria-describedby="react-select-8-placeholder"
                    aria-expanded="false"
                    aria-haspopup="true"
                    autocapitalize="none"
                    autocomplete="off"
                    autocorrect="off"
                    class=""
                    id="react-select-8-input"
                    role="combobox"
                    spellcheck="false"
                    style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                    tabindex="0"
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class=" css-1hb7zxy-IndicatorsContainer"
              >
                <span
                  class=" css-1u9des2-indicatorSeparator"
                />
                <div
                  aria-hidden="true"
                  class=" css-1xc3v61-indicatorContainer"
                >
                  <svg
                    aria-hidden="true"
                    class="css-tj5bde-Svg"
                    focusable="false"
                    height="20"
                    viewBox="0 0 20 20"
                    width="20"
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="chakra-form-control css-1dx7k9g"
          role="group"
        >
          <label
            class="chakra-form__label css-uvho5g"
            for="field-:rh:"
            id="field-:rh:-label"
          >
            Age
          </label>
          <input
            class="chakra-input css-124be3e"
            id="field-:rh:"
            type="number"
            value=""
          />
        </div>
        <div
          class="chakra-form-control css-1p8wjmg"
          role="group"
        >
          <label
            class="chakra-form__label css-u9c2cp"
            for="field-:ri:"
            id="field-:ri:-label"
          >
            Somatic type
          </label>
          <div
            class=" css-b62m3t-container"
          >
            <span
              class="css-1f43avz-a11yText-A11yText"
              id="react-select-9-live-region"
            />
            <span
              aria-atomic="false"
              aria-live="polite"
              aria-relevant="additions text"
              class="css-1f43avz-a11yText-A11yText"
            />
            <div
              class=" css-13cymwt-control"
            >
              <div
                class=" css-1fdsijx-ValueContainer"
              >
                <div
                  class=" css-1jqq78o-placeholder"
                  id="react-select-9-placeholder"
                >
                  Select...
                </div>
                <div
                  class=" css-qbdosj-Input"
                  data-value=""
                >
                  <input
                    aria-autocomplete="list"
                    aria-describedby="react-select-9-placeholder"
                    aria-expanded="false"
                    aria-haspopup="true"
                    autocapitalize="none"
                    autocomplete="off"
                    autocorrect="off"
                    class=""
                    id="react-select-9-input"
                    role="combobox"
                    spellcheck="false"
                    style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                    tabindex="0"
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class=" css-1hb7zxy-IndicatorsContainer"
              >
                <span
                  class=" css-1u9des2-indicatorSeparator"
                />
                <div
                  aria-hidden="true"
                  class=" css-1xc3v61-indicatorContainer"
                >
                  <svg
                    aria-hidden="true"
                    class="css-tj5bde-Svg"
                    focusable="false"
                    height="20"
                    viewBox="0 0 20 20"
                    width="20"
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="chakra-form-control css-1elsowl"
          role="group"
        >
          <label
            class="chakra-form__label css-u9c2cp"
            for="field-:rj:"
            id="field-:rj:-label"
          >
            Objective
          </label>
          <div
            class=" css-b62m3t-container"
          >
            <span
              class="css-1f43avz-a11yText-A11yText"
              id="react-select-10-live-region"
            />
            <span
              aria-atomic="false"
              aria-live="polite"
              aria-relevant="additions text"
              class="css-1f43avz-a11yText-A11yText"
            />
            <div
              class=" css-13cymwt-control"
            >
              <div
                class=" css-1fdsijx-ValueContainer"
              >
                <div
                  class=" css-1jqq78o-placeholder"
                  id="react-select-10-placeholder"
                >
                  Select...
                </div>
                <div
                  class=" css-qbdosj-Input"
                  data-value=""
                >
                  <input
                    aria-autocomplete="list"
                    aria-describedby="react-select-10-placeholder"
                    aria-expanded="false"
                    aria-haspopup="true"
                    autocapitalize="none"
                    autocomplete="off"
                    autocorrect="off"
                    class=""
                    id="react-select-10-input"
                    role="combobox"
                    spellcheck="false"
                    style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                    tabindex="0"
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class=" css-1hb7zxy-IndicatorsContainer"
              >
                <span
                  class=" css-1u9des2-indicatorSeparator"
                />
                <div
                  aria-hidden="true"
                  class=" css-1xc3v61-indicatorContainer"
                >
                  <svg
                    aria-hidden="true"
                    class="css-tj5bde-Svg"
                    focusable="false"
                    height="20"
                    viewBox="0 0 20 20"
                    width="20"
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="chakra-form-control css-0"
          role="group"
        >
          <label
            class="chakra-form__label css-u9c2cp"
            for="field-:rk:"
            id="field-:rk:-label"
          >
            Weight
          </label>
          <input
            class="chakra-input css-0"
            id="field-:rk:"
            type="number"
            value="0"
          />
        </div>
        <div
          class="chakra-form-control css-0"
          role="group"
        >
          <h2
            class="chakra-heading css-0"
          >
            Question 2
          </h2>
          <p
            class="chakra-text css-0"
          >
            On a scale of 1 to 10, how active are you on a day to day basis?
          </p>
          <div
            class="chakra-slider css-lnyvlk"
            style="position: relative; user-select: none; outline: 0; padding-top: 0px; padding-bottom: 0px;"
            tabindex="-1"
          >
            <div
              class="chakra-slider__track css-0"
              id="slider-track-:rm:"
              style="position: absolute; top: 50%; transform: translateY(-50%); width: 100%;"
            >
              <div
                class="chakra-slider__filled-track css-mzpod1"
                style="position: absolute; top: 50%; transform: translateY(-50%); width: 0%; left: 0%;"
              />
            </div>
            <div
              aria-orientation="horizontal"
              aria-valuemax="10"
              aria-valuemin="0"
              aria-valuenow="0"
              class="chakra-slider__thumb css-1hjihvm"
              id="slider-thumb-:rm:"
              role="slider"
              style="position: absolute; user-select: none; left: calc(0% - 0px);"
              tabindex="0"
            />
            <input
              type="hidden"
              value="0"
            />
          </div>
        </div>
        <div
          class="chakra-form-control css-0"
          role="group"
        >
          <h2
            class="chakra-heading css-0"
          >
            Question 3
          </h2>
          <p
            class="chakra-text css-0"
          >
            Are you doing any type of exercises?
          </p>
          <label
            class="chakra-checkbox css-1elxtiq"
          >
            <input
              aria-disabled="false"
              aria-invalid="false"
              class="chakra-checkbox__input"
              id="field-:rn:"
              style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
              type="checkbox"
              value="0"
            />
            <span
              aria-hidden="true"
              class="chakra-checkbox__control css-dnty2r"
            />
          </label>
        </div>
      </div>
      <button
        class="chakra-button css-4xx2wk"
        data-testid="calorie-button"
        type="button"
      >
        Calculate
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
